src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -198,6 +198,9 @@ module InternalUse =
     /// to be used as name for all namespaces that do not have a valid name
     let UnknownNamespace = "__UnknownNamespaceName__"
 
+    /// the implicit namespace in notebooks
+    let NotebookNamespace = "__NotebookNamespace__"
+
```

Comment:
Carrying this comment from @bettinaheim over so it doesn't get forgotten: https://github.com/microsoft/qsharp-compiler/pull/1457#discussion_r907866459

Reply:
We could have only one namespace name, but I'd want to rename it to something more neutral than "Unknown," like "Default" or "Main." But I think "Notebook" is probably least confusing...